includes/class-sensei-teacher.php

```diff
@@ -311,6 +312,28 @@ public function get_teachers_and_authors() {
 
 	}
 
+	/**
+	 * Get list of users who can author courses, lessons, and quizzes.
+	 * Optionally specify the fields to return from the DB.
+	 *
+	 * @since 3.13.4
+	 * @access private
+	 *
+	 * @param  string|array $fields Fields to return from DB. Defaults to 'id'.
+	 * @return array
+	 */
+	private function get_teachers_and_authors_with_fields( $fields = 'id' ) {
```

Comment:
The id change would also apply here (and in the PHPDoc above).

Reply:
✅